src/main/python/ttconv/tt.py

```diff
@@ -376,7 +376,7 @@ def convert(args):
     #
     # Write out the converted file
     #
-    tree_from_model.write(outputfile, encoding="utf-8")
```

Comment:
The XML declaration is optional in XML 1.0, which is the [default](https://www.w3.org/TR/ttml2/#concrete-encoding) for IMSC and TTML.

Reply:
Well, I really don't know about the standards, I check the ttml with https://github.com/skynav/ttt
The result is that, without the xml declaration, I get encoding errors on some diacritical marks and french symbols (ç ...).
Is utf-8 the default when no xml declaration exists -> do you think it is a ttt issue which fails to defaults to utf-8 ? or should the particular encoding characters be detected to trigger the xml declaration ?

Reply:
@ngaullier Can you provide a sample file that fails TTV validation? Looking at code I wrote that uses TTV, I force UTF-8:

```
      args.add("--force-encoding");
      args.add("UTF-8");
```

There is also a known issue with BOM (https://github.com/skynav/ttt/issues/193) but it is probably not relevant here.

Reply:
[samplefrench.zip](https://github.com/sandflow/ttconv/files/13294150/samplefrench.zip)
PYTHONPATH=src/main/python python3 src/main/python/ttconv/tt.py convert -i /mnt/d/samplefrench.stl -o /mnt/d/samplefrench.ttml
>Timed Text Verifier (TTV) [7.2-SNAPSHOT] Copyright (c) 2013-21 Skynav, Inc.
[E]:Malformed US-ASCII at byte offset 815 of one byte.

On "é" of métisse:
10:00:45:06-10:00:47:07    SGN.SN.EBN.CS.VP=00.0012.FF.00.20C
   	[DOUBLE HEIGHT][ALPHA CYAN]          <<qui est métisse,>>|| (30)

Yes, I also experienced ttv/BOM issue, but personnaly I don't like BOMs anyway!